wid/l2cap.py

```diff
@@ -310,6 +310,11 @@ def hdl_wid_60(params: WIDParams):
     return False
 
 
+def hdl_wid_61(_: WIDParams):
+    """description: Please confirm the IUT does not send the L2CAP Data to the Upper Tester."""
```

Comment:
hmm shouldn't we actually check this before confirming ?

Reply:
to me, it looks like , they want to inform IUT side to not send anything. I tested this with new wid, otherwise giving missing wid error

Reply:
@ilhanates We discussed this offline and we can merge as is. We will create an issue to add verification mechanism in this and other mmi handlers.